src/com/company/Console.java

```diff
@@ -1,18 +1,38 @@
 package com.company;
-
+import  java.lang.String;
 import java.util.List;
+
 import java.util.Scanner;
 
 public class Console {
-
     public void showConsole() {
         System.out.println("Choose action:");
         System.out.println("1. Add new user to list:");
         System.out.println("2. Remove person from list:");
         System.out.println("3. Show all users in list:");
+        System.out.println("4. Change object name in list:");
         System.out.println("0. Stop the program:");
     }
 
+    public List<People> change_object(List<People> population) {
+        Scanner scanner = new Scanner((System.in));
+        try {
+            System.out.println("Enter new object name:");
+            String login = scanner.nextLine();
+            System.out.println("Enter variable object index:");
+            int index = scanner.nextInt();
+            population.get(index).setLogin(login);
```

Comment:
Please, try to avoid such contractions.
In this case it may be like workaround, but in general its better to use `population.set(index, newPeople)`